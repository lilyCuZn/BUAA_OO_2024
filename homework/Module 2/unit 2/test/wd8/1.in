[1.1]59-FROM-7-TO-10
[1.4]RESET-DCElevator-3-5-7-0.6
[2.8]18-FROM-11-TO-1
[3.5]37-FROM-2-TO-9
[3.9]63-FROM-11-TO-8
[4.0]3-FROM-6-TO-3
[4.1]38-FROM-3-TO-10
[5.3]21-FROM-5-TO-6
[5.4]49-FROM-1-TO-11
[6.5]20-FROM-4-TO-10
[8.5]9-FROM-11-TO-1
[8.9]32-FROM-8-TO-6
[11.2]28-FROM-4-TO-11
[11.8]14-FROM-5-TO-7
[12.1]19-FROM-3-TO-9
[12.5]RESET-DCElevator-1-9-5-0.5
[13.4]35-FROM-10-TO-2
[13.6]5-FROM-8-TO-3
[14.3]4-FROM-5-TO-9
[14.5]50-FROM-10-TO-8
[16.7]44-FROM-9-TO-10
[18.8]69-FROM-9-TO-6
[20.2]43-FROM-7-TO-2
[22.3]RESET-DCElevator-4-6-6-0.6
[23.7]22-FROM-2-TO-11
[24.0]58-FROM-1-TO-5
[24.1]39-FROM-3-TO-5
[25.1]RESET-Elevator-6-3-0.4
[25.3]67-FROM-6-TO-11
[25.9]RESET-DCElevator-5-9-6-0.3
[26.2]33-FROM-11-TO-1
[26.3]47-FROM-6-TO-3
[26.6]RESET-DCElevator-2-8-4-0.2
[27.8]52-FROM-2-TO-7
[27.8]24-FROM-2-TO-1
[27.8]70-FROM-6-TO-4
[28.0]53-FROM-5-TO-4
[28.3]56-FROM-11-TO-7
[28.5]7-FROM-7-TO-1
[28.7]23-FROM-8-TO-11
[29.0]55-FROM-4-TO-9
[29.6]62-FROM-3-TO-2
