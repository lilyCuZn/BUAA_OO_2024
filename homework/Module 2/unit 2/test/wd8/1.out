[   1.2470]RECEIVE-59-3
[   1.6480]ARRIVE-2-3
[   2.0490]ARRIVE-3-3
[   2.4490]ARRIVE-4-3
[   2.8500]ARRIVE-5-3
[   2.9440]RECEIVE-18-4
[   3.2500]ARRIVE-6-3
[   3.3440]ARRIVE-2-4
[   3.6480]RECEIVE-37-2
[   3.6500]ARRIVE-7-3
[   3.6510]OPEN-7-3
[   3.7450]ARRIVE-3-4
[   3.8510]IN-59-7-3
[   4.0430]RECEIVE-63-6
[   4.0490]ARRIVE-2-2
[   4.0490]OPEN-2-2
[   4.0520]CLOSE-7-3
[   4.1420]RECEIVE-3-3
[   4.1450]ARRIVE-4-4
[   4.2420]RECEIVE-38-6
[   4.2490]IN-37-2-2
[   4.4430]ARRIVE-2-6
[   4.4500]CLOSE-2-2
[   4.4520]ARRIVE-8-3
[   4.5450]ARRIVE-5-4
[   4.8440]ARRIVE-3-6
[   4.8440]OPEN-3-6
[   4.8500]ARRIVE-3-2
[   4.8520]ARRIVE-9-3
[   4.9460]ARRIVE-6-4
[   5.0440]IN-38-3-6
[   5.2450]CLOSE-3-6
[   5.2500]ARRIVE-4-2
[   5.2520]ARRIVE-10-3
[   5.2530]OPEN-10-3
[   5.3460]ARRIVE-7-4
[   5.4430]RECEIVE-21-4
[   5.4530]OUT-59-10-3
[   5.5420]RECEIVE-49-4
[   5.6450]ARRIVE-4-6
[   5.6510]ARRIVE-5-2
[   5.6530]CLOSE-10-3
[   5.7460]ARRIVE-8-4
[   6.0460]ARRIVE-5-6
[   6.0510]ARRIVE-6-2
[   6.0530]ARRIVE-9-3
[   6.1470]ARRIVE-9-4
[   6.4460]ARRIVE-6-6
[   6.4520]ARRIVE-7-2
[   6.4540]ARRIVE-8-3
[   6.5470]ARRIVE-10-4
[   6.6420]RECEIVE-20-3
[   6.8460]ARRIVE-7-6
[   6.8520]ARRIVE-8-2
[   6.8540]ARRIVE-7-3
[   6.9480]ARRIVE-11-4
[   6.9480]OPEN-11-4
[   7.1480]IN-18-11-4
[   7.2470]ARRIVE-8-6
[   7.2520]ARRIVE-9-2
[   7.2520]OPEN-9-2
[   7.2540]ARRIVE-6-3
[   7.2550]OPEN-6-3
[   7.3490]CLOSE-11-4
[   7.4530]OUT-37-9-2
[   7.4550]IN-3-6-3
[   7.6470]ARRIVE-9-6
[   7.6530]CLOSE-9-2
[   7.6550]CLOSE-6-3
[   7.7490]ARRIVE-10-4
[   8.0480]ARRIVE-10-6
[   8.0480]OPEN-10-6
[   8.0550]ARRIVE-5-3
[   8.1490]ARRIVE-9-4
[   8.2480]OUT-38-10-6
[   8.4480]CLOSE-10-6
[   8.4560]ARRIVE-4-3
[   8.5500]ARRIVE-8-4
[   8.6440]RECEIVE-9-1
[   8.8490]ARRIVE-11-6
[   8.8490]OPEN-11-6
[   8.8560]ARRIVE-3-3
[   8.8560]OPEN-3-3
[   8.9500]ARRIVE-7-4
[   9.0430]RECEIVE-32-4
[   9.0440]ARRIVE-2-1
[   9.0490]IN-63-11-6
[   9.0570]OUT-3-3-3
[   9.2500]CLOSE-11-6
[   9.2570]CLOSE-3-3
[   9.3510]ARRIVE-6-4
[   9.4440]ARRIVE-3-1
[   9.6500]ARRIVE-10-6
[   9.6570]ARRIVE-4-3
[   9.6570]OPEN-4-3
[   9.7510]ARRIVE-5-4
[   9.8450]ARRIVE-4-1
[   9.8580]IN-20-4-3
[  10.0500]ARRIVE-9-6
[  10.0580]CLOSE-4-3
[  10.1510]ARRIVE-4-4
[  10.2450]ARRIVE-5-1
[  10.4510]ARRIVE-8-6
[  10.4510]OPEN-8-6
[  10.4580]ARRIVE-5-3
[  10.5520]ARRIVE-3-4
[  10.6450]ARRIVE-6-1
[  10.6510]OUT-63-8-6
[  10.8510]CLOSE-8-6
[  10.8590]ARRIVE-6-3
[  10.9520]ARRIVE-2-4
[  11.0460]ARRIVE-7-1
[  11.2590]ARRIVE-7-3
[  11.3440]RECEIVE-28-1
[  11.3520]ARRIVE-1-4
[  11.3530]OPEN-1-4
[  11.4460]ARRIVE-8-1
[  11.5530]OUT-18-1-4
[  11.6590]ARRIVE-8-3
[  11.7530]CLOSE-1-4
[  11.7540]OPEN-1-4
[  11.8470]ARRIVE-9-1
[  11.9420]RECEIVE-14-5
[  11.9540]IN-49-1-4
[  12.0600]ARRIVE-9-3
[  12.1540]CLOSE-1-4
[  12.2420]RECEIVE-19-3
[  12.2470]ARRIVE-10-1
[  12.3430]ARRIVE-2-5
[  12.4600]ARRIVE-10-3
[  12.4600]OPEN-10-3
[  12.5540]ARRIVE-2-4
[  12.6470]ARRIVE-11-1
[  12.6470]OPEN-11-1
[  12.6610]OUT-20-10-3
[  12.7430]ARRIVE-3-5
[  12.8480]IN-9-11-1
[  12.8610]CLOSE-10-3
[  12.9550]ARRIVE-3-4
[  13.0480]CLOSE-11-1
[  13.1430]ARRIVE-4-5
[  13.2610]ARRIVE-9-3
[  13.3550]ARRIVE-4-4
[  13.4480]ARRIVE-10-1
[  13.5430]RECEIVE-35-6
[  13.5440]ARRIVE-5-5
[  13.5440]OPEN-5-5
[  13.6620]ARRIVE-8-3
[  13.7420]RECEIVE-5-3
[  13.7440]IN-14-5-5
[  13.7550]ARRIVE-5-4
[  13.7560]OPEN-5-4
[  13.8490]ARRIVE-9-1
[  13.9430]ARRIVE-9-6
[  13.9440]CLOSE-5-5
[  13.9560]IN-21-5-4
[  14.0620]ARRIVE-7-3
[  14.1560]CLOSE-5-4
[  14.2490]ARRIVE-8-1
[  14.3440]ARRIVE-10-6
[  14.3440]OPEN-10-6
[  14.3440]ARRIVE-6-5
[  14.4420]RECEIVE-4-6
[  14.4620]ARRIVE-6-3
[  14.5440]IN-35-10-6
[  14.5560]ARRIVE-6-4
[  14.5570]OPEN-6-4
[  14.6420]RECEIVE-50-1
[  14.6490]ARRIVE-7-1
[  14.7440]CLOSE-10-6
[  14.7450]ARRIVE-7-5
[  14.7450]OPEN-7-5
[  14.7570]OUT-21-6-4
[  14.8630]ARRIVE-5-3
[  14.9450]OUT-14-7-5
[  14.9570]CLOSE-6-4
[  15.0500]ARRIVE-6-1
[  15.1450]ARRIVE-9-6
[  15.1450]CLOSE-7-5
[  15.2630]ARRIVE-4-3
[  15.3570]ARRIVE-7-4
[  15.4500]ARRIVE-5-1
[  15.5450]ARRIVE-8-6
[  15.6630]ARRIVE-3-3
[  15.6630]OPEN-3-3
[  15.7580]ARRIVE-8-4
[  15.8500]ARRIVE-4-1
[  15.8640]IN-19-3-3
[  15.9460]ARRIVE-7-6
[  16.0640]CLOSE-3-3
[  16.1580]ARRIVE-9-4
[  16.2510]ARRIVE-3-1
[  16.3460]ARRIVE-6-6
[  16.4640]ARRIVE-4-3
[  16.5590]ARRIVE-10-4
[  16.6510]ARRIVE-2-1
[  16.7460]ARRIVE-5-6
[  16.8440]RECEIVE-44-3
[  16.8650]ARRIVE-5-3
[  16.9590]ARRIVE-11-4
[  16.9590]OPEN-11-4
[  17.0510]ARRIVE-1-1
[  17.0520]OPEN-1-1
[  17.1470]ARRIVE-4-6
[  17.1590]OUT-49-11-4
[  17.2520]OUT-9-1-1
[  17.2650]ARRIVE-6-3
[  17.3600]CLOSE-11-4
[  17.4520]CLOSE-1-1
[  17.5470]ARRIVE-3-6
[  17.6650]ARRIVE-7-3
[  17.7600]ARRIVE-10-4
[  17.8530]ARRIVE-2-1
[  17.9520]ARRIVE-2-6
[  17.9520]OPEN-2-6
[  18.0660]ARRIVE-8-3
[  18.1530]OUT-35-2-6
[  18.1600]ARRIVE-9-4
[  18.2530]ARRIVE-3-1
[  18.3530]CLOSE-2-6
[  18.4660]ARRIVE-9-3
[  18.4660]OPEN-9-3
[  18.5610]ARRIVE-8-4
[  18.5610]OPEN-8-4
[  18.6530]ARRIVE-4-1
[  18.6540]OPEN-4-1
[  18.6670]OUT-19-9-3
[  18.6670]IN-44-9-3
[  18.7540]ARRIVE-3-6
[  18.7610]IN-32-8-4
[  18.8540]IN-28-4-1
[  18.8670]CLOSE-9-3
[  18.9440]RECEIVE-69-4
[  18.9610]CLOSE-8-4
[  19.0540]CLOSE-4-1
[  19.1540]ARRIVE-4-6
[  19.2670]ARRIVE-10-3
[  19.2680]OPEN-10-3
[  19.3620]ARRIVE-7-4
[  19.4550]ARRIVE-5-1
[  19.4680]OUT-44-10-3
[  19.5540]ARRIVE-5-6
[  19.5550]OPEN-5-6
[  19.6680]CLOSE-10-3
[  19.7550]IN-4-5-6
[  19.7620]ARRIVE-6-4
[  19.7620]OPEN-6-4
[  19.8550]ARRIVE-6-1
[  19.9550]CLOSE-5-6
[  19.9620]OUT-32-6-4
[  20.0680]ARRIVE-9-3
[  20.1630]CLOSE-6-4
[  20.2550]ARRIVE-7-1
[  20.3430]RECEIVE-43-4
[  20.3560]ARRIVE-6-6
[  20.4690]ARRIVE-8-3
[  20.4690]OPEN-8-3
[  20.5630]ARRIVE-7-4
[  20.6570]ARRIVE-8-1
[  20.6690]IN-5-8-3
[  20.7560]ARRIVE-7-6
[  20.8700]CLOSE-8-3
[  20.9630]ARRIVE-8-4
[  21.0570]ARRIVE-9-1
[  21.1560]ARRIVE-8-6
[  21.2700]ARRIVE-7-3
[  21.3640]ARRIVE-9-4
[  21.3640]OPEN-9-4
[  21.4570]ARRIVE-10-1
[  21.5570]ARRIVE-9-6
[  21.5570]OPEN-9-6
[  21.5640]IN-69-9-4
[  21.6700]ARRIVE-6-3
[  21.7570]OUT-4-9-6
[  21.7640]CLOSE-9-4
[  21.8580]ARRIVE-11-1
[  21.8580]OPEN-11-1
[  21.9570]CLOSE-9-6
[  22.0580]OUT-28-11-1
[  22.0700]ARRIVE-5-3
[  22.1640]ARRIVE-8-4
[  22.2580]CLOSE-11-1
[  22.4710]ARRIVE-4-3
[  22.5650]ARRIVE-7-4
[  22.5650]OPEN-7-4
[  22.6590]ARRIVE-10-1
[  22.6590]OPEN-10-1
[  22.7650]IN-43-7-4
[  22.8590]IN-50-10-1
[  22.8710]ARRIVE-3-3
[  22.8710]OPEN-3-3
[  22.9650]CLOSE-7-4
[  23.0590]CLOSE-10-1
[  23.0710]OUT-5-3-3
[  23.2720]CLOSE-3-3
[  23.3660]ARRIVE-6-4
[  23.3660]OPEN-6-4
[  23.4600]ARRIVE-9-1
[  23.5660]OUT-69-6-4
[  23.7660]CLOSE-6-4
[  23.8440]RECEIVE-22-1
[  23.8600]ARRIVE-8-1
[  23.8600]OPEN-8-1
[  24.0600]OUT-50-8-1
[  24.1420]RECEIVE-58-5
[  24.1670]ARRIVE-5-4
[  24.2420]RECEIVE-39-3
[  24.2420]OPEN-3-3
[  24.2610]CLOSE-8-1
[  24.4430]IN-39-3-3
[  24.5430]ARRIVE-6-5
[  24.5670]ARRIVE-4-4
[  24.6430]CLOSE-3-3
[  24.6610]ARRIVE-7-1
[  24.9430]ARRIVE-5-5
[  24.9670]ARRIVE-3-4
[  25.0430]ARRIVE-4-3
[  25.0610]ARRIVE-6-1
[  25.2430]RESET_ACCEPT-6-3-0.4
[  25.2430]RESET_BEGIN-6
[  25.3430]ARRIVE-4-5
[  25.3670]ARRIVE-2-4
[  25.3680]OPEN-2-4
[  25.4440]ARRIVE-5-3
[  25.4440]OPEN-5-3
[  25.4620]ARRIVE-5-1
[  25.5680]OUT-43-2-4
[  25.6440]OUT-39-5-3
[  25.7440]ARRIVE-3-5
[  25.7680]CLOSE-2-4
[  25.8450]CLOSE-5-3
[  25.8620]ARRIVE-4-1
[  26.1440]ARRIVE-2-5
[  26.2620]ARRIVE-3-1
[  26.4440]RESET_END-6
[  26.4440]RECEIVE-67-6
[  26.4440]RECEIVE-33-1
[  26.4440]RECEIVE-47-1
[  26.5440]ARRIVE-1-5
[  26.5440]OPEN-1-5
[  26.6620]ARRIVE-2-1
[  26.6630]OPEN-2-1
[  26.7450]IN-58-1-5
[  26.8630]IN-22-2-1
[  26.9450]CLOSE-1-5
[  27.0630]CLOSE-2-1
[  27.3450]ARRIVE-2-5
[  27.4630]ARRIVE-3-1
[  27.7450]ARRIVE-3-5
[  27.8640]ARRIVE-4-1
[  27.9430]RECEIVE-52-4
[  27.9430]RECEIVE-24-2
[  27.9440]OPEN-2-4
[  27.9440]RECEIVE-70-2
[  28.1420]RECEIVE-53-5
[  28.1440]IN-52-2-4
[  28.1460]ARRIVE-4-5
[  28.2640]ARRIVE-5-1
[  28.3440]ARRIVE-8-2
[  28.3440]CLOSE-2-4
[  28.4420]RECEIVE-56-1
[  28.5460]ARRIVE-5-5
[  28.5460]OPEN-5-5
[  28.6420]RECEIVE-7-5
[  28.6640]ARRIVE-6-1
[  28.7440]ARRIVE-7-2
[  28.7440]ARRIVE-3-4
[  28.7460]OUT-58-5-5
[  28.8420]RECEIVE-23-4
[  28.9460]CLOSE-5-5
[  29.0650]ARRIVE-7-1
[  29.1420]RECEIVE-55-3
[  29.1440]ARRIVE-6-2
[  29.1440]OPEN-6-2
[  29.1450]ARRIVE-4-4
[  29.3450]IN-70-6-2
[  29.3470]ARRIVE-6-5
[  29.4650]ARRIVE-8-1
[  29.5430]ARRIVE-4-3
[  29.5430]OPEN-4-3
[  29.5450]ARRIVE-5-4
[  29.5450]CLOSE-6-2
[  29.7430]RECEIVE-62-3
[  29.7430]IN-55-4-3
[  29.7470]ARRIVE-7-5
[  29.7470]OPEN-7-5
[  29.8650]ARRIVE-9-1
[  29.9430]CLOSE-4-3
[  29.9450]ARRIVE-6-4
[  29.9450]ARRIVE-5-2
[  29.9470]IN-7-7-5
[  30.1470]CLOSE-7-5
[  30.2650]ARRIVE-10-1
[  30.3430]ARRIVE-5-3
[  30.3450]ARRIVE-7-4
[  30.3450]OPEN-7-4
[  30.3460]ARRIVE-4-2
[  30.3460]OPEN-4-2
[  30.5460]OUT-52-7-4
[  30.5460]OUT-70-4-2
[  30.5480]ARRIVE-6-5
[  30.6660]ARRIVE-11-1
[  30.6660]OPEN-11-1
[  30.7440]ARRIVE-6-3
[  30.7460]CLOSE-7-4
[  30.7460]CLOSE-4-2
[  30.8660]OUT-22-11-1
[  30.9480]ARRIVE-5-5
[  30.9480]OPEN-5-5
[  31.0660]CLOSE-11-1
[  31.0660]OPEN-11-1
[  31.1440]ARRIVE-7-3
[  31.1460]ARRIVE-8-4
[  31.1460]OPEN-8-4
[  31.1470]ARRIVE-3-2
[  31.1480]IN-53-5-5
[  31.2660]IN-33-11-1
[  31.2670]IN-56-11-1
[  31.3460]IN-23-8-4
[  31.3480]CLOSE-5-5
[  31.4670]CLOSE-11-1
[  31.5440]ARRIVE-8-3
[  31.5460]CLOSE-8-4
[  31.5470]ARRIVE-2-2
[  31.5470]OPEN-2-2
[  31.7470]IN-24-2-2
[  31.7480]ARRIVE-4-5
[  31.7480]OPEN-4-5
[  31.8670]ARRIVE-10-1
[  31.9440]ARRIVE-9-3
[  31.9440]OPEN-9-3
[  31.9460]ARRIVE-9-4
[  31.9470]CLOSE-2-2
[  31.9490]OUT-53-4-5
[  32.1450]OUT-55-9-3
[  32.1490]CLOSE-4-5
[  32.2670]ARRIVE-9-1
[  32.3450]CLOSE-9-3
[  32.3470]ARRIVE-10-4
[  32.3480]ARRIVE-1-2
[  32.3480]OPEN-1-2
[  32.5480]OUT-24-1-2
[  32.5490]ARRIVE-3-5
[  32.6670]ARRIVE-8-1
[  32.7450]ARRIVE-8-3
[  32.7470]ARRIVE-11-4
[  32.7470]OPEN-11-4
[  32.7480]CLOSE-1-2
[  32.9470]OUT-23-11-4
[  32.9490]ARRIVE-2-5
[  33.0680]ARRIVE-7-1
[  33.0680]OPEN-7-1
[  33.1450]ARRIVE-7-3
[  33.1470]CLOSE-11-4
[  33.2680]OUT-56-7-1
[  33.3490]ARRIVE-1-5
[  33.3490]OPEN-1-5
[  33.4680]CLOSE-7-1
[  33.5460]ARRIVE-6-3
[  33.5490]OUT-7-1-5
[  33.7500]CLOSE-1-5
[  33.8680]ARRIVE-6-1
[  33.8690]OPEN-6-1
[  33.9460]ARRIVE-5-3
[  34.0690]IN-47-6-1
[  34.2690]CLOSE-6-1
[  34.3460]ARRIVE-4-3
[  34.6690]ARRIVE-5-1
[  34.7460]ARRIVE-3-3
[  34.7470]OPEN-3-3
[  34.9470]IN-62-3-3
[  35.0690]ARRIVE-4-1
[  35.1470]CLOSE-3-3
[  35.4700]ARRIVE-3-1
[  35.4700]OPEN-3-1
[  35.5470]ARRIVE-2-3
[  35.5470]OPEN-2-3
[  35.6700]OUT-47-3-1
[  35.7470]OUT-62-2-3
[  35.8700]CLOSE-3-1
[  35.9480]CLOSE-2-3
[  36.2700]ARRIVE-2-1
[  36.6710]ARRIVE-1-1
[  36.6710]OPEN-1-1
[  36.8710]OUT-33-1-1
[  37.0710]CLOSE-1-1
